src/transport/MixTransport.cpp

```diff
@@ -50,8 +51,80 @@ double MixTransport::thermalConductivity()
 
 void MixTransport::getThermalDiffCoeffs(double* const dt)
 {
-    for (size_t k = 0; k < m_nsp; k++) {
-        dt[k] = 0.0;
+    update_T();
+    update_C();
+    if (!m_bindiff_ok) {
+        updateDiff_T();
+    }
+    if (!m_viscwt_ok) {
+        updateViscosity_T();
+    }
+
+    const std::vector<double>& M = m_thermo->molecularWeights();
+    const double* y = m_thermo->massFractions();
+
+    std::vector<double>& a = m_spwork;
+    std::vector<double>& x = m_spwork2;
+    m_thermo->getMoleFractions(x.data());
+
+    for (size_t k=0; k<m_nsp; ++k) {
+        dt[k] = 0.;
+        if (x[k] < Tiny)
+        {
+            a[k] = 0.;
+            continue;
+        }
+
+        double lambda_mono_k = (15./4.) * m_visc[k] / M[k];
+
+        double sum = 0.;
+        for(size_t j=0; j<m_nsp; ++j) {
```

Comment:
Need a space after `for`.
```suggestion
        for (size_t j=0; j<m_nsp; ++j) {
```